src/bytecode/concrete.py

```diff
@@ -439,24 +541,46 @@ def to_code(
         if stacksize is None:
             stacksize = self.compute_stacksize(check_pre_and_post=check_pre_and_post)
 
-        return types.CodeType(
-            self.argcount,
-            self.posonlyargcount,
-            self.kwonlyargcount,
-            nlocals,
-            stacksize,
-            int(self.flags),
-            code_str,
-            tuple(self.consts),
-            tuple(self.names),
-            tuple(self.varnames),
-            self.filename,
-            self.name,
-            self.first_lineno,
-            lnotab,
-            tuple(self.freevars),
-            tuple(self.cellvars),
-        )
+        if sys.version_info >= (3, 11):
+            return types.CodeType(
+                self.argcount,
+                self.posonlyargcount,
+                self.kwonlyargcount,
+                nlocals,
+                stacksize,
+                int(self.flags),
+                code_str,
+                tuple(self.consts),
+                tuple(self.names),
+                tuple(self.varnames),
+                self.filename,
+                self.name,
+                "",  # XXX qualname
```

Comment:
```suggestion
                self.qualname
```
with `self.qualname = self.name` in `BaseBytecode.__init__`

Reply:
We need to get the actual qualname when disassembling and I will take care of that later